config/templates/cloud-config-controller

```diff
@@ -693,6 +705,66 @@ write_files:
            $kubectl uncordon "$hostname"
            echo "done."'
 
+  {{if .Experimental.Plugins.Rbac.Enabled }}
+
+  #https://github.com/coreos/kube-aws/issues/230#issuecomment-274946933
+  - path: /srv/kubernetes/rbac/clusterrole.yaml
+    content: |
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1alpha1
+        metadata:
+            name: full-access
+        rules:
+          - apiGroups: ["*"]
+            resources: ["*"]
+            verbs: ["*"]
+          - nonResourceURLs: ["*"]
+            verbs: ["*"]
+
```

Comment:
I've noticed recent kube-aws 1.5.1 clusters come bootstrapped with a 'cluster-admin' cluster role that is the same as the 'full-access' above. Referencing that could reduce this patch. On the other hand, it is nice that the RBAC in those patch is self-contained and without implicit dependencies.